qcfractal/cli/qcfractal_manager.py

```diff
@@ -119,12 +120,61 @@ class Config(SettingsCommonConfig):
         extra = "allow"
 
 
+class ParslExecutorSettings(BaseSettings):
+    address: str = None
+
+    def __init__(self, **kwargs):
+        """Enforce that the keys we are going to set remain untouched"""
+        # This set blocks the Parsl Executor and Provider keywords which we set, so the names of the keywords align
+        # to those classes' kwargs, not whatever Fractal chooses to use as keywords.
+        forbidden_set = {"label", "provider", "cores_per_worker", "max_workers"}
+        bad_set = set(kwargs.keys()) & forbidden_set
+        if bad_set:
+            raise KeyError("The following items were set as part of parsl executor, however, "
+                           "there are other config items which control these in more generic "
+                           "settings locations: {}".format(bad_set))
+        super().__init__(**kwargs)
+
+    class Config(SettingsCommonConfig):
+        # This overwrites the base config to allow other keywords to be fed in
+        extra = "allow"
+
+
+class ParslProviderSettings(BaseSettings):
+    partition: str = None
+
+    def __init__(self, **kwargs):
+        """Enforce that the keys we are going to set remain untouched"""
+        # This set blocks the Parsl Executor and Provider keywords which we set, so the names of the keywords align
+        # to those classes' kwargs, not whatever Fractal chooses to use as keywords.
+        forbidden_set = {"nodes_per_block", "max_blocks", "worker_init", "scheduler_options", "wall_time"}
+        bad_set = set(kwargs.keys()) & forbidden_set
+        if bad_set:
+            raise KeyError("The following items were set as part of parsl's provider, however, "
+                           "there are other config items which control these in more generic "
+                           "settings locations: {}".format(bad_set))
+        super().__init__(**kwargs)
```

Comment:
We may want to have a class at some point that does this for us if we keep repeating these operations.